ci-operator/config/openshift/openshift-tests-private/.claude/agents/create_YZ_stream_upgrade_jobs.md

```diff
@@ -0,0 +1,33 @@
+---
+name: create_YZ_stream_upgrade_jobs
```

Comment:
suggestion, what about a simple name instead? The upgrade jobs are Y/Z streams so naming as `YZ_stream` is redundant. what about `create_upgrade_jobs`?

Reply:
OK, nit: be consistent with your naming pattern, following snake cases. YZ -> yz. They don't have to be capitalized in file names